collectd/30_statsd.conf.erb

```diff
@@ -1,5 +1,5 @@
 <Plugin statsd>
-  Host "127.0.0.1"
+  Host "0.0.0.0"
```

Comment:
the system firewall will prevent this being open the world, right?

Reply:
Definitely. But docker containers can't hit it if it's not listening on other interfaces.